src/components/landing/codeChallenge/Leaderboard.tsx

```diff
@@ -0,0 +1,162 @@
+import * as React from "react";
+import WorkshopsHeader from "../../workshops/WorkshopsHeader";
+import BackButton from "../../shared/BackButton";
+
+const Leaderboard: React.FC<{}> = (props) => {
+  return (
+    <div
+      className={
+        "bg-gradient-to-b from-neutral-800 to-neutral-900 p-5 min-h-screen"
+      }
+    >
+      <BackButton link={"/"} />
+      <div className={"max-w-5xl mx-auto my-5 mt-10 px-0 md:px-10"}>
+        <h3 className={"text-xl text-orange-500 font-bold font-Poppins"}>
+          Code Challenge Leaderboard
```

Comment:
Increase the font size here, and make its color consistent with that in the workshops page. I would recommend reusing the styles from the `WorkshopsHeader` component

[https://github.com/miguelaenlle/jhhs-hack-club/blob/main/src/components/workshops/WorkshopsHeader.tsx](https://github.com/miguelaenlle/jhhs-hack-club/blob/main/src/components/workshops/WorkshopsHeader.tsx)